Documentation/LanguageServer/MinGW.md

```diff
@@ -1,6 +1,43 @@
 For developers using MinGW on Windows, we recommend you start with the following **c_cpp_properties.json** template.  Select "C/Cpp: Edit Configurations" from the command palette to create this file if you haven't already.
 
-Note that you may have to change the MinGW version number to match what you have installed. Eg. `C:/MinGW/lib/gcc/mingw32/5.3.0/` instead of `C:/MinGW/lib/gcc/mingw32/6.3.0/`.
+## With extension version 0.16.1 and higher:
+
+Starting with version 0.16.1, if you set `compilerPath` and `intelliSenseMode`, you no longer need to copy the system include path or defines to `includePath` and `defines` to enable IntelliSense to work properly. However, `browse.path` still needs to be updated manually to add the system include paths to enable code browsing for system headers. For example:
+
+```json
+{
+    "configurations": [
+        {
```

Comment:
Can you add the cppStandard, cStandard? It's recommended to set those correctly too.

Reply:
We have reasonable defaults for those (and I think the extension will automatically add them for you once you set the compilerPath) so I didn't think it was necessary to talk about it.  The less information we have to convey to make things work, the better, IMO.

Reply:
Ok